caps/charts.py

```diff
@@ -0,0 +1,92 @@
+from typing import TYPE_CHECKING
+
+from .models import Council, DataPoint
+
+import altair as alt
+import pandas as pd
+from charting import ChartBundle
+
+
+def multi_emissions_chart(council: "Council"):
```

Comment:
Think can import the future annotations thing and take the Council out of the string